### PR TITLE
Added new variable name mapping for v2

### PR DESCRIPTION
**Pull request by howelj**
## Pre-review checklist

I have ensured that:

- [ ] Code runs successfully in the development environment.
- [ ] New code is tested to prove that it works. If not, justify reasoning.
- [ ] New code is documented using [type hinting](https://realpython.com/lessons/type-hinting/) and following the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) docstring format.
- [x] New code follows the project naming standards.

## Post-review checklist

As I merge these changes, I have:

- [ ] Updated the related Jira ticket, including approximate time spent (including review).
- [ ] Moved the related Jira ticket to Done.

### CODE REVIEW

**Comment by howelj** [comment]
New branch with naming map for v2, this will be tested in following tickets

File: cishouseholds/pipeline/input_variable_names.py [hunk 1]
```diff
@@ -33,3 +33,231 @@
     "Date Samples Received Oxford": "blood_sample_received_date",
     "Voyager Date Created": "blood_sample_collected_datetime",
 }
+
+
+v2_variable_name_map = {
+    "raw_variable_name": "pipeline_variable_name",
```
Comment (osborj) [inline cishouseholds/pipeline/input_variable_names.py:39]: Don't think this is required

File: cishouseholds/pipeline/input_variable_names.py [hunk 2]
```diff
@@ -33,3 +33,231 @@
     "Date Samples Received Oxford": "blood_sample_received_date",
     "Voyager Date Created": "blood_sample_collected_datetime",
 }
+
+
+v2_variable_name_map = {
```
Comment (osborj) [inline cishouseholds/pipeline/input_variable_names.py:38]: Change to iqvia_v2_variable_name_map

**Review by osborj — COMMENTED** [review]
Change according to comments

File: cishouseholds/pipeline/input_variable_names.py (hunk 1 above)
Comment (howelj) [reply cishouseholds/pipeline/input_variable_names.py]: Removed this line

File: cishouseholds/pipeline/input_variable_names.py (hunk 2 above)
Comment (howelj) [reply cishouseholds/pipeline/input_variable_names.py]: Renamed as stated

**Review by osborj — APPROVED** [review]

**Pull request merged**